src/NuGet.Core/NuGet.Build.Tasks.Console/MSBuildStaticGraphRestore.cs

```diff
@@ -606,6 +607,8 @@ private DependencyGraphSpec GetDependencyGraphSpec(string entryProjectPath, IDic
                     // Get the PackageSpecs in parallel because creating each one is relatively expensive so parallelism speeds things up
                     Parallel.ForEach(projects, new ParallelOptions { MaxDegreeOfParallelism = Environment.ProcessorCount }, project =>
                     {
+                        MSBuildLogger.LogMinimal($"For {project.OuterProject.FullPath}," +
```

Comment:
@nkolev92 This seems to spam the console, even with minimal logging is specified. Did you mean to leave this in there?

Reply:
Not on purpose. 
I'll fix it.